hack/release/build/cross.sh

```diff
@@ -48,3 +48,6 @@ export GOOS=linux GOARCH=arm64
 export GOOS=linux GOARCH=ppc64le
 EOF
 )
+
+# add sha256 for binaries
+for f in "${REPO_ROOT}"/bin/kind-*; do shasum -a 256 "$f" > "$f".sha256sum; done
```

Comment:
don't use the oneliner and keep the last line, is easier to read
```suggestion
for f in "${REPO_ROOT}"/bin/kind-*; do 
    shasum -a 256 "$f" > "$f".sha256sum
done

```

Reply:
Updated

Reply:
hmm this adds the whole path to the generated file, I think that we have to cd first in the folder
```
cd "${REPO_ROOT}"/bin
for f in kind-*; do
      shasum -a 256 "$f" > "$f".sha256sum
done
```

Reply:
yup it was adding full path. I have updated that to change to the directory and then run sha command.